src/config/index.types.d.ts

```diff
@@ -0,0 +1,25 @@
+type QuizQuestionId =
```

Comment:
Shouldn't this be an `as const` object like we generally do for enums.

Reply:
yeah I could do that. will do as part of another PR, more work is needed for the earn use case as we're expending it.